spec/index.bs

```diff
@@ -3765,16 +3765,16 @@ interface GPUQueue {
     void signal(GPUFence fence, GPUFenceValue signalValue);
 
     void writeBuffer(
-        ArrayBuffer sourceData,
-        GPUSize64 sourceOffset,
-        GPUBuffer destination,
-        GPUSize64 destinationOffset,
-        GPUSize64 size);
+        GPUBuffer buffer,
+        GPUSize64 bufferOffset,
+        ArrayBuffer data,
+        optional GPUSize64 dataOffset = 0,
+        optional GPUSize64 size);
```

Comment:
doesn't it have to have = 0 at the end?

Reply:
No it's not required to have a default value. Kai commented the following:

> Size should probably be optional with no default, so zero-size copies are really zero-size.

Which I think makes sense.